cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -33,7 +33,7 @@ import com.here.ort.model.config.ScannerConfiguration
 import com.here.ort.model.mapper
 import com.here.ort.model.readValue
 import com.here.ort.scanner.LocalScanner
-import com.here.ort.scanner.ScanResultsCache
```

Comment:
how about naming this 'Repository' after the 'Repository pattern, see https://de.wikipedia.org/wiki/Repository_(Entwurfsmuster)
also in clean architecture this is a common name, https://proandroiddev.com/clean-architecture-data-flow-dependency-rule-615ffdd79e29

Reply:
In our case, I believe the term "repository" to already be tight associated to the VCS context. Hence I'm fine with "storage", or even slight prefer it.

Reply:
Ok, I'll keep "storage" then.